pymc3/distributions/continuous.py

```diff
@@ -1359,6 +1359,24 @@ def logp(value, a, b):
 
         return bound(logp, value >= 0, value <= 1, a > 0, b > 0)
 
+    def logcdf(value, a, b):
+        r"""
+        Compute the log of cumulative distribution function for the Kumaraswamy distribution
+        at the specified value.
+
+        Parameters
+        ----------
+        value: numeric or np.ndarray or aesara.tensor
+            Value(s) for which log CDF is calculated. If the log CDF for
+            multiple values are desired the values must be provided in a numpy
+            array or Aesara tensor.
+
+        Returns
+        -------
+        TensorVariable
+        """
+        return bound(at.log1p(-((1 - value ** a) ** b)), value >= 0, value <= 1)
```

Comment:
Value above one should give a logcdf of 0, so we need a switch: `tt.switch(value < 1, normal expression, 0)`, and to remove `value <= 1` as a bound condition. We should still keep the `a > 0` and `b > 0` conditions as in the `logp`

Reply:
does `tt` equal to `at`? Because I assume `tt` is Theano tensor and it is not available.

Reply:
Yes of course. It was a mistake, I am still used to writing with theano's abbreviation :p